tools/update_dlint.ts

```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env -S deno run --allow-write --allow-read --allow-net
-// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+#!/usr/bin/env -S deno run --allow-write --allow-read --allow-net --allow-env
```

Comment:
I added `--allow-env` because access to these environment variables was requested:

```console
┏ ⚠️  Deno requests env access to "GRACEFUL_FS_PLATFORM".
┠─ Learn more at: https://docs.deno.com/go/--allow-env
┠─ Run again with --allow-env to bypass this prompt.
┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all env permissions) > 
┏ ⚠️  Deno requests env access to "TEST_GRACEFUL_FS_GLOBAL_PATCH".
┠─ Learn more at: https://docs.deno.com/go/--allow-env
┠─ Run again with --allow-env to bypass this prompt.
┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all env permissions) > 
┏ ⚠️  Deno requests env access to "READABLE_STREAM".
┠─ Learn more at: https://docs.deno.com/go/--allow-env
┠─ Run again with --allow-env to bypass this prompt.
┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all env permissions) > 
```